docs/STANDALONE_MODE.md

```diff
@@ -0,0 +1,30 @@
+# Standalone mode
```

Comment:
The flag currently is `--stand-alone` with the hyphen. In this document `standalone` is used. These should match. For usability I think the flag should change to one word, but for ease, it makes more sense to just change the documentation to match the flag.

Reply:
I don't think that it is good idea to change now `--stand-alone` flag to `--standalone` because some of our plugins have been update to new versions of plugin libs to provide diagnostics information. I think that in documentation I can change "stanalone mode" to "stand-alone mode".  @kjlyon what do you think? 

In the future if someone decide to change flag he/she should also take into account that it is needed to update all plugins which use old versions of snap-plugin-libs.

Reply:
I agree.